_source/_posts/2022-10-12-oktane-for-developers.md

```diff
@@ -0,0 +1,77 @@
+﻿---
+layout: blog_post
+title: "Hello Developers, Time to Get Your Oktane On"
+author: havi-hoffman
+by: advocate
+communities: [security] 
+description: "Oktane 2022 happens in San Francisco from November 8-10 and we'd love to meet you there at the Developer Hub. We've designed presentations and workshops with developers in mind." 
+tags: [announcement, conference] 
+tweets: 
+- ""
+- ""
+- ""
+image: 
```

Comment:
Should this have a social image? Maybe we can use one of the ones that marketing uses to advertise Oktane? For instance: https://twitter.com/okta/status/1580026976865517569/photo/1